RefrigeratorExe/RefrigeratorExe/Shelf.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RefrigeratorExe
+{
+    internal class Shelf :IComparable<Shelf>
+    {
+        private int _floor;
+        private int _space;
+        public int ShelfSpace = 20;
+        public Guid Id{ get; }
+        public int Floor {
+            get
+            {
+                return _floor;
+            }
+            private set
+            {
+                if (value >= 0)
+                    _floor = value;
+                else
+                    throw new ArithmeticException("Illegal floor ,floor must be a positive number");
+            }
+        }
+        public int Space {
+            get
+            {
+                return _space;
+            }
+            private set
+            {
+                if (value >= 0)
+                    _space = value;
+                else
+                    throw new ArithmeticException("Illegal space , space must be a positive number");
+            }
+        }
+        public List<Item> Items;
+        public Shelf(int floor) 
```

Comment:
Like my comments to the other classes - here is the final properties and constructor as should be:

public Guid Id{ get; }
public int Floor { get; }
public int Space { get; set; }
public List<Item> Items;
public const int ShelfSpace = 20;

public Shelf(int floor)
{
    if (floor <= 0)
        throw new ArgumentException("Must be positive value", $"{floor}");

    Id = Guid.NewGuid();
    Space = ShelfSpace;
    Floor = floor;
    Items = new List<Item>();
}